deployments/chef/example_attrs.json

```diff
@@ -1,5 +1,5 @@
 {"signalfx_agent": {
-  "package_stage": "final",
+  "package_stage": "release",
   "agent_version": "4.3.1",
```

Comment:
Is agent 4.3.1 going to be in the release stage of ARtifactory?

Reply:
> Is agent 4.3.1 going to be in the release stage of ARtifactory?

Yes, all rpms and debs have been migrated to artifactory and re-signed.